apps/gruppetto/pages/_app.tsx

```diff
@@ -142,6 +143,8 @@ function CustomApp({Component, pageProps}: CustomAppProps) {
               <title key="title">{siteTitle}</title>
               <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
+              <GoogleAnalytics gaId="G-FL5CRKBXV4" />
```

Comment:
Probably shouldn't be in the <Head> according to documentation. But I guess it doesn't hurt either.

Reply:
You are probably right. Moved it below the content.